### ISSUE TITLE
Quick delete - app Alma bug. 

### ISSUE THREAD

**Issue by robynschofield** [ExLibrisGroup/barcode-fast-withdraw#2]
We were trying to delete items (using the delete app) from one of our library locations where the item is back to its permanent location but when we do this we get a message such as Could not be withdrawn – This item is on a reading list for course S-EDUC4024-22X08-SP2,S-EDUC4024-22X07-SP2. Which other items were but that item wasn't). I reported this to ExLibris and they think that it is a bug with the app, they stated, using the Quick Delete app, the parameter Override Indicates whether the item should be deleted even if warnings exist. Optional. By default: false. So, even after selecting "Override" it does not allow to delete via the app. This looks like a bug in the app itself.

**Comment by ori229** [comment]
I'll take a look. What was the case number?

**Comment by robynschofield** [comment]
Hi ??

Do you mean the Exlibris case number?

Regards Robyn

**Comment by ori229** [comment]
Yes, please. (I work in Ex Libris, and I would like to have a look at the case to have some background)

**Comment by robynschofield** [comment]
06755492 - Delete an item that the bibliographic record is attached to a reading list as a citation.

**Comment by ori229** [comment]
We were not able to reproduce (on a local QA server that has data similar to yours).
Perhaps you have a barcode of an item (which the app doesn't delete, even with "override") that we can try to delete on your production or sandbox?

![image](https://github.com/ExLibrisGroup/barcode-fast-withdraw/assets/139582/cb35ddb0-0554-4743-96ec-d7df85f8beaa)

![image](https://github.com/ExLibrisGroup/barcode-fast-withdraw/assets/139582/1a7e1852-2a61-4243-a818-888f4e120df2)

**Comment by kumarravnesh** [comment]
Hi @ori229 

We've noticed when deleting items it appears that the app is not deleting the associated holding record (if last copy) nor the bib record (if last copy).

Ex Libris Case: 08233997

Thanks
Rav
